Instagram/Controllers/FeedViewController.swift

```diff
@@ -8,11 +8,86 @@
 import UIKit
 
 /// Feed screen.
-class FeedViewController: UIViewController {
-
+final class FeedViewController: UIViewController {
+  
+  // MARK: - Private enum.
+  private enum Items {
```

Comment:
```suggestion
  private enum CelType {
```
item? Саня ты в порядке?)